### ISSUE TITLE
Separate Scroll on Admin Nav Pane

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#2573]
When the main content of an admin screen is overly long (see PHP Info), having the admin navigation pane acquire an independent scroll becomes a huge convenience.

See the MySQL Docs site for an example:
https://dev.mysql.com/doc/refman/8.0/en/

**Comment by bhsmither** [comment]
It would also be nice to have the row of tabs stick at the top of the screen

**Comment by bhsmither** [comment]
To get the sticky tabs:
```
admin template common.breadcrumb.php, line 1
From:
<div id="breadcrumbs">
To:
<div id="breadcrumbs" style="position:sticky;top:0;z-index:9">
```
```
admin template main.php, lines 50-51:
From:
         <div id="content">
            <div id="tab_control">
To:
         <div id="content">
            <div style="position:sticky;top:26px;height:10px;z-index:9;background-color:#ECECEC;"></div>
            <div id="tab_control" style="position:sticky;top:36px;z-index:9;background-color:#ECECEC;margin-top:0;">

```

**Comment by abrookbanks** [comment]
The sticky tab control looked like this with that code. 
![Screenshot 2020-09-10 134043](https://user-images.githubusercontent.com/4520803/92730179-4d2bfc80-f36b-11ea-8be5-df34e6f944f3.jpg)

**Comment by bhsmither** [comment]
I am experimenting on why your Google Chart is not hiding its overflow that is obscuring the Nav pane. Either not hiding it, or the Chart is badly positioned and scaled too large.

For example: where are your Quick Tasks? The "WARNING" (no Close button) and "Dashboard" line up and are flush against the Nav pane.

I think there may be an additional or missing `</div>` tag somewhere messing with `#content_body`.

**Comment by bhsmither** [comment]
In my Firefox, I can make changes to the CSS and HTML against the page being displayed. I did this on the CubeCart Demo site.

On the Demo site, I've discovered that once the new line following `<div id="content">` is added, switching panels via clicking on the tabs is no longer possible. I do not yet know why.

On my dev site, when I use the Developer Tools to *remove* the extra line and style from `<div id="tab_control">`, switching panes cannot be done.

So, I think moving, or applying/removing styling in this way - using the Tools - confuses this javascript that handles tab pane switching.
